addOns/automation/CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 ## Unreleased
 ### Added
 - Import Job profile (Issue 7078).
+- URL presence tests in automation framework
```

Comment:
I'd suggest `Add test type to check for the presence of a URL.` (The `in automation framework` is redundant, the release notes are in this add-on).